tools/dm-task-configuration-file-intro.md

```diff
@@ -9,166 +9,139 @@ category: tools
-                                                                           # `loader` at the same time.
-        syncer-config-name: "global"                                       # The Syncer configuration name. You cannot set it and 
-                                                                           # `syncer` at the same time.
+        mydumper-config-name: "global"                                     # The mydumper configuration name.
+        loader-config-name: "global"                                       # The Loader configuration name.
+        syncer-config-name: "global"                                       # The Syncer configuration name.
 
     -
-        config:
-            host: "192.168.199.118"
-            port: 5306
-            user: "root"
-            password: "1234"
-        instance-id: "instance118-5306"
-
-        mydumper:                                                          # The mydumper configuration. You cannot set it and 
-                                                                           # `mydumper-config-name` at the same time.
-            mydumper-path: "./mydumper"                                    # The mydumper binary file path. It is generated by 
-                                                                           # Ansible deployment application and needs no 
-                                                                           # configuration.
-            threads: 4
-            chunk-filesize: 8
-            skip-tz-utc: true
-            extra-args: "-B test -T t1,t2"
-    
-        loader:                                                            # The Loader configuration. You cannot set it and 
-                                                                           # `loader-config-name` at the same time.
-            pool-size: 32                                                  # The number of threads that execute mydumper SQL 
-                                                                           # files concurrently in Loader
-            dir: "./dumped_data"
-    
-        syncer:                                                            # The Syncer configuration. You cannot set it and 
-                                                                           # `syncer-config-name` at the same time.
-            worker-count: 32                                               # The number of threads that synchronize binlog events 
-                                                                           # concurrently in Syncer
-            batch: 2000
-            max-retry: 200
+        source-id: "mysql-replica-02"                                      # The ID of the upstream instance or replication group. It can be configured by referring to the `source_id` in the `inventory.ini` file or the `source-id` in the `dm-master.toml` file.
```

Comment:
```suggestion
        source-id: "mysql-replica-02"                                      # The ID of the upstream instance or replica group. It can be configured by referring to the `source_id` in the `inventory.ini` file or the `source-id` in the `dm-master.toml` file.
```